ncd-calculator/src/__test__/labelManager.test.ts

```diff
@@ -0,0 +1,77 @@
+import {LabelManager} from "@/functions/labelUtils.ts";
+
+describe('Label Manager', () => {
+	let labelManager: LabelManager;
+	beforeEach(() => {
+		labelManager = LabelManager.getInstance();
+		labelManager.clear();
+	})
+	
+	test('should properly register and retrieve labels', () => {
+		labelManager.registerLabel('id1', 'Display Label 1');
+		expect(labelManager.getDisplayLabel('id1')).toBe('Display Label 1');
+	})
+	
+	
+	test('should handle empty or undefined inputs gracefully', () => {
+		labelManager.registerLabel('', 'Empty ID');
+		expect(labelManager.getDisplayLabel('')).toBeUndefined();
+		
+		labelManager.registerLabel(undefined, 'Undefined ID');
+		expect(labelManager.getDisplayLabel(undefined)).toBeUndefined();
```

Comment:
what is the purpose of this?

Reply:
Just to check some invalid input would result in `undefined` or error.